src/application/controllers/event-controller/event-controller.ts

```diff
@@ -41,4 +47,52 @@ export class EventController {
       });
     }
   }
+
+  public static async addMessageEvent(
+    req: Request,
+    res: Response
+  ): Promise<void> {
+    emitWarning("a");
+    try {
+      const codiEvent = req.body.codi;
+      console.log("A");
+      const chatEvent: IChat = await EventRepository.getChatEvent(codiEvent);
+      console.log("A");
+      const newMessage: IMessage = await MessageRepository.addMessage(req.body.content, req.body.userId, req.body.data);
+      console.log("A");
+      const chat: Chat = await ChatRepository.addMessage(chatEvent, newMessage);
+      console.log("A");
+      await EventRepository.modifyChatEvent(codiEvent, chat);
+      res.status(200);
+      res.json({ 
+        message: 'chat sent it',
+        messages: newMessage,
+      });
+    } catch (e) {
+      res.status(500);
+      res.json({
+        error: e,
+      });
+    }
+  }
+
+  public static async getAllMessages(
+    req: Request,
+    res: Response
+  ): Promise<void> {
+    try {
+      const codiEvent = req.body.codi;
+      const chatEvent: IChat = await EventRepository.getChatEvent(codiEvent);
+      const messages: IMessage[] = await ChatRepository.getMessages(chatEvent);
```

Comment:
Si el getChatEvent ya hace populate de messages, te podria quedar algo asi:
```suggestion
      const chatEvent: IChat = await EventRepository.getChatEvent(codiEvent);
      const messages = chatEvent.messages;
```